docs/community/working-group.md

```diff
@@ -0,0 +1,189 @@
+# Working Groups
+
+[Working Groups (WGs)][wg] are collections of contributors with knowledge of specific components or
+technologies in the PowerShell domain.
+They are responsible for issue triage/acceptance, code reviews, and providing their expertise to
+others in issues, PRs, and RFC discussions.
```

Comment:
This is repeating a section above, should one of those sections just link to the other so we have a single definition? I fear them getting out of sync.